pkg/security/tests/module_tester.go

```diff
@@ -966,24 +962,19 @@ func newTestModule(t testing.TB, macroDefs []*rules.MacroDefinition, ruleDefs []
 	}
 
 	var cmdWrapper cmdWrapper
-	if testEnvironment == DockerEnvironment {
+	if testEnvironment == DockerEnvironment || ebpfLessEnabled {
 		cmdWrapper = newStdCmdWrapper()
 	} else {
-		if opts.staticOpts.enableEBPFLess {
-			// docker not supported by ebpf less
-			cmdWrapper = newStdCmdWrapper()
+		wrapper, err := newDockerCmdWrapper(st.Root(), st.Root(), "ubuntu")
+		if err == nil {
+			cmdWrapper = newMultiCmdWrapper(wrapper, newStdCmdWrapper())
 		} else {
-			wrapper, err := newDockerCmdWrapper(st.Root(), st.Root(), "ubuntu")
-			if err == nil {
-				cmdWrapper = newMultiCmdWrapper(wrapper, newStdCmdWrapper())
-			} else {
-				// docker not present run only on host
-				cmdWrapper = newStdCmdWrapper()
-			}
+			// docker not present run only on host
+			cmdWrapper = newStdCmdWrapper()
```

Comment:
is this changes have to be removed/adpated? as they already were part of another PR (#22192)

Reply:
It just simplifies the logic

Reply:
ok right, I looked at it too fast :) sorry